src/main/java/org/kiwiproject/dropwizard/util/metrics/ServerLoadFetcher.java

```diff
@@ -54,9 +52,8 @@ public ServerLoadFetcher() {
     public Optional<String> get() {
         var process = processes.launch("uptime");
 
-        try (var reader = new BufferedReader(new InputStreamReader(process.getInputStream(), UTF_8))) {
-            var output = reader.readLine();
-
+        try {
+            var output = KiwiIO.readInputStreamOf(process);
```

Comment:
This would also work:

```suggestion
            var output = KiwiIO.readLinesFromInputStreamOf(process);
            var firstLine = first(output);
            var matcher = LOAD_AVERAGE_PATTERN.matcher(firstLine);
```

Is it "better"?

Reply:
Since the original code works "as is", and is less code, let's go with that